### PR TITLE
Add image registries into container config

### PR DESCRIPTION
**Pull request by ganeshmaharaj**
Fixes: #67

Signed-off-by: Ganesh Maharaj Mahalingam <ganesh.mahalingam@intel.com>

### CODE REVIEW

**Comment by krsna1729** [comment]
@ganeshmaharaj Failed testing it
```
    clr-01: cat: invalid option -- 'c'
    clr-01: Try 'cat --help' for more information.
==> clr-01: An error occurred. The error will be shown after all tasks complete.
    clr-03: Calling post-update helper scripts.
    clr-03: Successfully installed 2 bundles
    clr-03: cat: invalid option -- 'c'
    clr-03: Try 'cat --help' for more information.
```
Probably need "EOF" for the first part

File: clr-k8s-examples/setup_system.sh [hunk 1]
```diff
@@ -10,6 +10,14 @@ ADD_NO_PROXY+=",$(hostname -I | sed 's/[[:space:]]/,/g')"
 sudo -E swupd update
 sudo -E swupd bundle-add cloud-native-basic storage-utils
 
+# Add registries for k8s onto crio's registries.conf
+if [ ! -f /etc/containers/registries.conf ]; then
+  cat << EORF > sudo bash -c "cat > /etc/containers/registries.conf"
```
Comment (krsna1729) [inline clr-k8s-examples/setup_system.sh]: @ganeshmaharaj Whats happening really here

Comment (ganeshmaharaj) [reply clr-k8s-examples/setup_system.sh]: should be fixed now!

**Comment by krsna1729** [comment]
@ganeshmaharaj Does not seem to solve the issue

```
Every 2.0s: kubectl get po --all-namespaces -owide                                                                                                              clr-01: Wed Mar 13 20:05:42 2019

NAMESPACE     NAME                              READY   STATUS         RESTARTS   AGE     IP                NODE     NOMINATED NODE   READINESS GATES
kube-system   canal-g96wb                       0/3     ErrImagePull   0          51s     192.168.121.178   clr-02   <none>           <none>
kube-system   canal-kv8tb                       2/3     Running        0          51s     192.168.121.42    clr-01   <none>           <none>
kube-system   canal-rgpmq                       0/3     ErrImagePull   0          51s     192.168.121.94    clr-03   <none>           <none>
```

**Comment by krsna1729** [comment]
Master
```
NAMESPACE     NAME                              READY   STATUS              RESTARTS   AGE   IP                NODE     NOMINATED NODE   READINESS GATES
kube-system   canal-cpsf5                       2/3     Running             0          37s   192.168.121.203   clr-01   <none>           <none>
kube-system   canal-nrrgf                       0/3     ImageInspectError   0          37s   192.168.121.24    clr-02   <none>           <none>
kube-system   canal-rn2r8                       0/3     ErrImagePull        0          37s   192.168.121.244   clr-03   <none>           <none>
```

**Comment by mcastelino** [comment]
We do not need this.

Root cause is https://github.com/clearlinux/cloud-native-setup/issues/70

**Pull request closed without merging**